### ISSUE TITLE
Babel error on npm start

### ISSUE THREAD

**Issue by msalsas** [ubilabs/react-geosuggest#114]
These errors are thrown when I run `npm start` after cloning the repository and executing `npm install`:

```
> browserify example/src/app.js -t babelify --extension=.jsx> example/dist/app.js

ReferenceError: [BABEL] /projectDir/react-geosuggest/example/src/app.js: Unknown option: /projectDir/react-geosuggest/.babelrc.presets while parsing file: /projectDir/react-geosuggest/example/src/app.js
    at Logger.error (/projectDir/react-geosuggest/node_modules/babelify/node_modules/babel-core/lib/transformation/file/logger.js:58:11)
    at OptionManager.mergeOptions (/projectDir/react-geosuggest/node_modules/babelify/node_modules/babel-core/lib/transformation/file/options/option-manager.js:126:29)
    at OptionManager.addConfig (/projectDir/react-geosuggest/node_modules/babelify/node_modules/babel-core/lib/transformation/file/options/option-manager.js:107:10)
    at OptionManager.findConfigs (/projectDir/react-geosuggest/node_modules/babelify/node_modules/babel-core/lib/transformation/file/options/option-manager.js:168:35)
    at OptionManager.init (/projectDir/react-geosuggest/node_modules/babelify/node_modules/babel-core/lib/transformation/file/options/option-manager.js:229:12)
    at File.initOptions (/projectDir/react-geosuggest/node_modules/babelify/node_modules/babel-core/lib/transformation/file/index.js:147:75)
    at new File (/projectDir/react-geosuggest/node_modules/babelify/node_modules/babel-core/lib/transformation/file/index.js:137:22)
    at Pipeline.transform (/projectDir/react-geosuggest/node_modules/babelify/node_modules/babel-core/lib/transformation/pipeline.js:164:16)
    at Babelify._flush (/projectDir/react-geosuggest/node_modules/babelify/index.js:28:24)
    at Babelify.<anonymous> (_stream_transform.js:118:12)

npm ERR! Linux 3.13.0-37-generic
npm ERR! argv "/usr/bin/nodejs" "/usr/bin/npm" "run" "js:example"
npm ERR! node v5.10.0
npm ERR! npm  v3.8.5
npm ERR! code ELIFECYCLE
npm ERR! react-geosuggest@1.19.0 js:example: `browserify example/src/app.js -t babelify --extension=.jsx> example/dist/app.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the react-geosuggest@1.19.0 js:example script 'browserify example/src/app.js -t babelify --extension=.jsx> example/dist/app.js'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the react-geosuggest package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     browserify example/src/app.js -t babelify --extension=.jsx> example/dist/app.js
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs react-geosuggest
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls react-geosuggest
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /projectDir/react-geosuggest/npm-debug.log

npm ERR! Linux 3.13.0-37-generic
npm ERR! argv "/usr/bin/nodejs" "/usr/bin/npm" "run" "prepare:example"
npm ERR! node v5.10.0
npm ERR! npm  v3.8.5
npm ERR! code ELIFECYCLE
npm ERR! react-geosuggest@1.19.0 prepare:example: `rm -rf example/dist && mkdir example/dist && npm run copy:assets && npm run js:example && npm run css`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the react-geosuggest@1.19.0 prepare:example script 'rm -rf example/dist && mkdir example/dist && npm run copy:assets && npm run js:example && npm run css'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the react-geosuggest package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     rm -rf example/dist && mkdir example/dist && npm run copy:assets && npm run js:example && npm run css
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs react-geosuggest
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls react-geosuggest
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /projectDir/react-geosuggest/npm-debug.log

npm ERR! Linux 3.13.0-37-generic
npm ERR! argv "/usr/bin/nodejs" "/usr/bin/npm" "run" "server"
npm ERR! node v5.10.0
npm ERR! npm  v3.8.5
npm ERR! code ELIFECYCLE
npm ERR! react-geosuggest@1.19.0 server: `npm run prepare:example && light-server -s example/dist -p 8000 -w 'src/*.jsx,example/src/*.js # npm run js:example' -w 'src/*.css,example/src/*.css # npm run css # reloadcss' -w 'example/src/*.html # npm run copy:assets # reload'`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the react-geosuggest@1.19.0 server script 'npm run prepare:example && light-server -s example/dist -p 8000 -w 'src/*.jsx,example/src/*.js # npm run js:example' -w 'src/*.css,example/src/*.css # npm run css # reloadcss' -w 'example/src/*.html # npm run copy:assets # reload''.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the react-geosuggest package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     npm run prepare:example && light-server -s example/dist -p 8000 -w 'src/*.jsx,example/src/*.js # npm run js:example' -w 'src/*.css,example/src/*.css # npm run css # reloadcss' -w 'example/src/*.html # npm run copy:assets # reload'
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs react-geosuggest
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls react-geosuggest
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /projectDir/react-geosuggest/npm-debug.log

npm ERR! Linux 3.13.0-37-generic
npm ERR! argv "/usr/bin/nodejs" "/usr/bin/npm" "start"
npm ERR! node v5.10.0
npm ERR! npm  v3.8.5
npm ERR! code ELIFECYCLE
npm ERR! react-geosuggest@1.19.0 start: `npm run server`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the react-geosuggest@1.19.0 start script 'npm run server'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the react-geosuggest package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     npm run server
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs react-geosuggest
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls react-geosuggest
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /projectDir/react-geosuggest/npm-debug.log

```

BTW `npm install` throws this warnings:

```
npm WARN optional Skipping failed optional dependency /chokidar/fsevents:
npm WARN notsup Not compatible with your operating system or architecture: fsevents@1.0.11
```

**Comment by msalsas** [comment]
It works if I remove `.babelrc`.

**Comment by ro-ka** [comment]
Solved by #111. Those `babel` upgrades…

Thanks for reporting!
